src/libs/state.js

```diff
@@ -857,7 +857,8 @@ const state = new Vue({
 
             let network = buffer.getNetwork();
             let isNewMessage = message.time >= buffer.last_read;
-            let isHighlight = Misc.mentionsNick(bufferMessage.message, network.ircClient.user.nick);
+            let isHighlight = !network ?
+                false : Misc.mentionsNick(bufferMessage.message, network.ircClient.user.nick);
```

Comment:
For consistency this should be structured as:
~~~javascript
            let isHighlight = !network ?
                false :
                Misc.mentionsNick(bufferMessage.message, network.ircClient.user.nick);
~~~